src/picker/PickerListBox.ts

```diff
@@ -0,0 +1,60 @@
+import * as React from "react";
+import { useForkRef } from "reakit-utils";
+import { createHook, createComponent } from "reakit-system";
+import {
+  CompositeHTMLProps,
+  CompositeOptions,
+  useComposite,
+  usePopover,
+} from "reakit";
+
+import { PICKER_STATE_KEYS } from "./__keys";
+import { usePortalShortcut } from "./__utils";
+import { PickerStateReturn } from "./PickerState";
+
+export type PickerOptions = CompositeOptions & PickerStateReturn;
+
+export type PickerHTMLProps = CompositeHTMLProps;
+
+export type PickerProps = PickerOptions & PickerHTMLProps;
+
+const usePickerListBox = createHook<PickerOptions, PickerHTMLProps>({
+  name: "usePickerListBox",
+  compose: [usePopover, useComposite],
+  keys: PICKER_STATE_KEYS,
+
+  useOptions(options, htmlProps) {
+    return options;
+  },
+
+  useProps(options, { ref: htmlRef, ...htmlProps }) {
+    const ref = React.useRef<HTMLElement>(null);
+
+    usePortalShortcut(ref, options);
+
+    React.useEffect(() => {
+      if (options.selectedValue && options.values) {
+        const selectedValue = options.values.find(
+          value => value.value === options.selectedValue,
+        );
+
+        if (selectedValue?.id) {
+          options.move?.(selectedValue.id);
+        }
+      }
+      // eslint-disable-next-line react-hooks/exhaustive-deps
+    }, [options.selectedValue, options.values, options.move, options.visible]);
```

Comment:
Not sure but this bit of code was causing a bug in our Select's Multiselect behaviour it was causing the focus to reset to the first item select everytime.

Reply:
But this code, just focuses the selected value when there is an selected value.
You can change the conditions, if you want to do the checks for your case.